packages/codegen/query/control.ts

```diff
@@ -43,15 +60,18 @@ export const CONTROL_FIELDS_FRAGMENT = gql`
           displayID
           name
           creationDate
+          displayID
         }
       }
     }
     subcontrols {
       totalCount
       edges {
         node {
+          id
           refCode
           description
+          displayID
```

Comment:
Don't think you'll need displayID for subcontrols
```suggestion
          displayID
```